src/main/java/com/indeed/jiraactions/api/response/issue/Issue.java

```diff
@@ -19,20 +28,61 @@
     public Field fields;
     public ChangeLog changelog;
 
+    // Multivalued rich object fields are treated different in Jira changelog histories than
+    //  multivalued primitive fields.
+    private static Set<String> MULTIVALUED_RICH_FIELDS = ImmutableSet.of(
+            "component"
```

Comment:
Only fixing component here and not fixing "fixversion" or any other fields is fine. But it's a good future enhancement. (Also we'll have to mess with the separator below since it might not always be "|").